specific/cmo_account/models/account.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    quote_ref_id = fields.Many2one(
+        'sale.order',
+        string='Quotation Ref.',
+        readonly=True,
+    )
+    quote_ref_number = fields.Char(
```

Comment:
As we already have quote_ref_id,
should this be related field, i..e, quote_ref_id.name ?